src/lib/swapper/swappers/ThorchainSwapper/getThorTradeQuote/getTradeQuote.ts

```diff
@@ -125,45 +142,27 @@ export const getThorTradeQuote = async (
     )
   }
 
-  const thorSwapStreamingSwaps = getConfig().REACT_APP_FEATURE_THOR_SWAP_STREAMING_SWAPS
+  const getRouteValues = (quote: ThornodeQuoteResponseSuccess, isStreaming: boolean) => ({
+    source: isStreaming ? THORCHAIN_STREAM_SWAP_SOURCE : SwapperName.Thorchain,
+    quote,
+    // expected receive amount after slippage (no affiliate_fee or liquidity_fee taken out of this value)
+    // TODO: slippage is currently being applied on expected_amount_out which is emit_asset - outbound_fee,
+    //       should slippage actually be applied on emit_asset?
+    expectedAmountOutThorBaseUnit: subtractBasisPointAmount(
+      quote.expected_amount_out,
+      quote.fees.slippage_bps,
+    ),
+    isStreaming,
+    estimatedExecutionTimeMs: quote.total_swap_seconds
```

Comment:
updated to use `total_swap_second` for both types of quote, the previous `inbound_confirmation_seconds + outbound_delay_seconds` are being calculated by thorchain: https://gitlab.com/thorchain/thornode/-/blob/develop/x/thorchain/querier_quotes.go#L699